### PR TITLE
Added latex command

### PR DESCRIPTION
**Pull request by sriram-sb**
Resolves Issue # 7

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import icon_animator
 import json
 import csv
+import sympy
```
Comment (BrandonZupan) [inline main.py]: Just import the functions that you use

File: main.py [hunk 2]
```diff
@@ -922,6 +923,14 @@ class RankEntry(Base):
...
     await ctx.send("Texas beat OU 48 to 45 in the Red River Rivalry with a last second field goal by Dicker the Kicker! :metal:")
 
+@client.command(name='latex')
+async def latexCommand(ctx, *args):
+    latex_input = "".join(args)
+    latex_input = "$${}$$".format(latex_input)
+    
+    sympy.printing.preview(latex_input, viewer='file', filename='latex_output.png')
+    await ctx.send(file=discord.File(open('latex_output.png', 'rb')))
+    os.remove('output.png')
```
Comment (BrandonZupan) [inline main.py]: This should be latex_output.png

**Review by BrandonZupan — CHANGES_REQUESTED** [review]
Couple things, only import what you need and fix the file name on like 933

File: main.py (hunk 2 above)
Comment (sriram-sb) [reply main.py]: Brain is fried 😅. Will do

Comment (sriram-b) [reply main.py]: Brain is fried 😅. Will do

File: main.py (hunk 1 above)
Comment (sriram-sb) [reply main.py]: Alright 👍

**Review by BrandonZupan — APPROVED** [review]
looks good thank you!

**Pull request merged**